_config.yml

```
@@ -12,6 +12,8 @@ color_scheme: dark
 aux_links:
   "Main Website":
     - "//vantibyte.com"
+  "Contact Us":
+    - "//vantibyte.com/contact/
```

You might be missing a quotation mark here!
```suggestion
    - "//vantibyte.com/contact/"
```